flixel/graphics/frames/FlxAtlasFrames.hx

```diff
@@ -212,18 +208,17 @@ class FlxAtlasFrames extends FlxFramesCollection
 	 * Internal method for LibGDX atlas parsing. It tries to extract dimensions info from specified string.
 	 *
 	 * @param   line   `String` to extract info from.
-	 * @param   size   `Array` to store extracted info to.
 	 * @return  `Array` filled with dimensions info.
 	 */
-	static function getDimensions(line:String, size:Array<Int>):Array<Int>
+	static function getDimensions(line:String):Array<Int>
```

Comment:
Might as well make this return a FlxPoint a this point?

Reply:
Or `{x: Int, y:Int}` if it's important that they're ints.

Reply:
Yes let's do that :+1: